### ISSUE TITLE
"Playback failed" error

### ISSUE THREAD

**Issue by ruuk** [plexinc/plex-for-kodi#175]
Not sure of the exact nature yet, getting 400 errors on DP . Waiting on some server logs.
Relevant forum thread: https://forums.plex.tv/discussion/298096/plex-kodi-playback-failed-error

**Comment by ruuk** [comment]
Have a possible fix. Awaiting verification via user testing.

**Comment by NOTORIOUSVR** [comment]
Is there an update to this issue?

The minute I set any sort of Bandwith limit I immediately get the Playback Failed error. set it back to Original (disabled) and playback works again.

The fix is to disable the "Limit remote stream bitrate" option on the server by setting it back to Original... 

So will a Dev please look into this ASAP, it only seems to effect the KODI ADD-ON when connected remotely. On Local LAN this issue doesn't exist (for obvious reasons).

Server ver: 1.12.3.4973
Plex add-on ver: 0.1.3
Kodi ver. 17.1

**Comment by ruuk** [comment]
Fixed

**Comment by NOTORIOUSVR** [comment]
This issue is not closed... Problem is just masked.

It now starts the video playback but again if you set the Limit remote stream bitrate option on the server it causes the playback to stutter like crazy and causes the add-on to become unstable/uncontrollable.

Disabling the bitrate option again makes everything smooth.

**Comment by ruuk** [comment]
The issue logged here is fixed. Feel free to open a new issue and provide logs and info and I'll look into it. 

Keep in mind that limiting bitrate will likely force a transcode that will increase server load, possibly causing the issues you describe.

**Comment by NOTORIOUSVR** [comment]
I will open a new issue then... This is not a transcoding issue from my server.

Thanks,
